### PR TITLE
add reproducibility as an s2i advantage

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@openshift/team-documentation ptal

File: architecture/core_concepts/builds_and_image_streams.adoc [hunk 1]
```diff
@@ -106,6 +106,10 @@ their application build.
 Ecosystem:: S2I encourages a shared ecosystem of images where you can leverage
 best practices for your applications.
 
+Reproducibility:: Produced images can include all inputs used to produce the image, 
+ensuring the image can be reproduced precisely including the versions of build tools 
+and dependencies.
+
```
Comment (tnguyen-rh) [inline architecture/core_concepts/builds_and_image_streams.adoc]: Suggest: "Produced images can include all inputs, including specific versions of build tools and dependencies.  This ensures that the image can be reproduced precisely."

**Comment by tnguyen-rh** [comment]
@bparees Small rewording suggested, then i'm happy to merge.

**Comment by bparees** [comment]
@tnguyen-rh done, thanks

**Comment by adellape** [comment]
[rev_history]
|link:../architecture/core_concepts/builds_and_image_streams.html[Core Concepts -> Builds and Image Streams]
|Added *Reproducibility* to the list of S2I advantages in the link:../architecture/core_concepts/builds_and_image_streams.html#source-build[Source-to-Image (S2I) Build] section.
%

**Pull request merged**